### DISCUSSION TITLE
ensure metrics all pass but run fails

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#3350] [Artillery]
The metrics I defined all passed, but the run overall failed and I don't see why. What else do I need to do ?

**Comment by hassy** [comment]
We're taking a look. In the meantime could you check the CloudWatch log for the worker that ran that test to see if there are any errors reported there?

**Reply by hassy** [reply]
@zoe-jobson-IL we've released an update to the dashboard to show Artillery CLI's exit code under Errors if it's not 0 (which would mark the test as failing). The CloudWatch logs might have an error message in them to explain what happened exactly.

We also just published an update to the docs to describe Artillery's use of exit codes: https://www.artillery.io/docs/reference/cli/exit-codes
